src/tests/ftest/util/server_utils.py

```diff
@@ -1195,7 +1195,7 @@ def search_log(self, pattern):
             result = run_remote(self.log, host, f"grep -E '{pattern}' {' '.join(log_files)}")
             for data in result.output:
                 if data.returncode == 0:
-                    matches = re.findall(fr'{pattern}', data.stdout)
+                    matches = re.findall(fr'{pattern}', str(data.stdout))
```

Comment:
FYI that `data.stdout` is a list of output lines. E.g. `str(list)` looks like:
```
>>> str(['foo', 'bar'])
"['foo', 'bar']"
```
A more proper search is probably to convert to a single string of output:
```suggestion
                    matches = re.findall(fr'{pattern}', '\n'.join(data.stdout))
```

Reply:
Sure, @phender mentioned that he is not going to add such kind of check and trying to use different approach, i am not sure if this PR is needed or not. or possibly reasonable for a quick fix.

@phender would you clarity it here?

Reply:
@daltonbohning Feel free to help refresh if we still want to land this simple fix.